BCH-Wunder-Project-team-4/Wunder.io
# article and case issues solved

## Link to ticket:

#74 + Case fade problem 

## Link to feature environment:

http://localhost:3000/cases/trimble-vision-plan-action
http://localhost:3000/all-articles/exploring-drupal-trends-wunderer-takeaways-drupalcon-lille-2023

## Changes proposed in this PR:

[Explain what was done and why. You can also add screenshots here if it helps.]

## How to test:

1. `git fetch origin issue74`
2. `git checkout issue74`
3. Check any article. Now there is the space between author and date 
4. Check any Case. Now all paragraphs there fades in
